modules/c/nrt/include/nrt/Utils.h

```diff
@@ -99,7 +107,8 @@ NRTAPI(void) nrt_Utils_decimalToGeographic(double decimal, int *degrees,
 
 /*!
  *  Convert the geographic coordinates (i.e., DMS) into decimal
- *  degrees as a double.
+ *  degrees as a double. The sign of the value is given by the largest
+ *  nonzero DMS value.
```

Comment:
Rather than "largest nonzero" I think "most significant nonzero" would be a little more clear (so that someone doesn't think largest in terms of literally biggest number".  Also would be good to add a sentence stating that all subsequent values should be positive.